services/frontend/.env.production

```diff
@@ -0,0 +1,8 @@
+REACT_APP_GRAPHQL_HTTP_URL=https://graphql.eosrate.io/v1/graphql
+REACT_APP_API_URL=https://bp.cryptolions.io
```

Comment:
I think we can also use our own api 

https://api.eosio.cr

Reply:
added!